tool/src/main/java/org/wildfly/security/tool/ElytronToolMessages.java

```diff
@@ -401,6 +400,36 @@
     @Message(id = NONE, value = "The relative or absolute path to the users file.")
     String cmdFileSystemRealmUsersFileDesc();
 
+    @Message(id = NONE, value = "The relative or absolute path to the credential store file that contains the secret key.")
+    String cmdFileSystemEncryptCredentialStoreDesc();
+
+    @Message(id = NONE, value = "The alias of the secret key stored in the credential store file.")
+    String cmdFileSystemEncryptSecretKeyDesc();
+
+    @Message(id = NONE, value = "Should a credential store be created? Set to false by default.")
```

Comment:
Looks like `create` defaults to `true` in WildFly so we should do the same here. This can be reworded to:

"Whether or not the credential store should be dynamically created if it doesn't exist. Set to true by default"